hc api: updated possible problems (#3482)
ydb-platform/ydb: ydb/docs/ru/core/reference/ydb-sdk/health-check-api.md

* `Pool usage over 90/95/99%` — один из CPU пулов перегружен.
* `System tablet is unresponsive / response time over 1000ms/5000ms` — системная таблетка не отвечает или отвечает долго.
* `Tablets are restarting too often` — таблетки слишком часто перезапускаются.
* `Tablets/Followers are dead` — таблетки не запущены (или не могут быть запущены).

> * `Node is restarting too often`,

Reviewer: ```suggestion
* `Node is restarting too often` — узлы слишком часто перезапускаются.
```

> * `The number of node restarts has increased`: количетство рестартов ноды превысило порог.

Reviewer: ```suggestion
* `The number of node restarts has increased` — количетство рестартов ноды превысило порог.
```

> * `The number of node restarts has increased` — количетство рестартов ноды превысило порог.

Reviewer 2: ```suggestion
* `The number of node restarts has increased` — количество рестартов ноды превысило порог.
```
Опечатка

* `Group has no redundancy` — группа хранения потеряла избыточность.
* `Group failed` — группа хранения потеряла целостность.
* `Group degraded` — в группе недоступно допустимое число дисков.

> * `The nodes have a time difference of ... ms` — возможные проблемы с координацией распределённых транзакций.

Reviewer 2: Я бы написал, что это расхождение времени на узлах, а дальше уже про проблемы

Reviewer: ```suggestion
* `The nodes have a time difference of ... ms` — расхождение времени на узлах, что может приводить к возможным проблемам с координацией распределённых транзакций.
```
